lib/env-validator.ts

```
@@ -0,0 +1,116 @@
+      this.errors.push('ADMIN_PASSWORD must be at least 12 characters long')
+    }
+    
+    if (process.env.CRON_SECRET && process.env.CRON_SECRET.length < 32) {
+      this.errors.push('CRON_SECRET must be at least 32 characters long')
+    }
+    
+    if (process.env.WORKER_AUTH_KEY && process.env.WORKER_AUTH_KEY.length < 32) {
+      this.errors.push('WORKER_AUTH_KEY must be at least 32 characters long')
+    }
+    
+    this.validated = true
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)